CRM/Contact/BAO/Query.php

```diff
@@ -4336,6 +4336,8 @@ public static function getQuery($params = NULL, $returnProperties = NULL, $count
    * @param string $apiEntity
    *   The api entity being called.
    *   This sort-of duplicates $mode in a confusing way. Probably not by design.
+   * @param bool $useGroupBy
+   *   Should the query be grouped by contact id?
```

Comment:
I think to the extent that Civi is consistent on something like this it is calling boolean values $isTest $isGroupBy etc